tools/UpdatePaths/Scripts/76783_penguins.txt

```diff
@@ -0,0 +1,4 @@
+/mob/living/simple_animal/pet/penguin : /mob/living/basic/pet/penguin{@OLD}
+/mob/living/simple_animal/pet/penguin/emperor : /mob/living/basic/pet/penguin/emperor{@OLD}
+/mob/living/simple_animal/pet/penguin/emperor/shamebrero : /mob/living/basic/pet/penguin/emperor/shamebrero{@OLD}
+/mob/living/simple_animal/pet/penguin/baby/permanent : /mob/living/basic/pet/penguin/baby/permanent{@OLD}
```

Comment:
the permanent subtype didn't exist on `simple_animal` before this, right? remove the subtype and then revert your map changes, then re-run the script.